src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -38,6 +36,8 @@ trait GoogleServicesDAO {
   def getToken(userInfo: UserInfo): Future[Option[String]]
   def getTokenDate(userInfo: UserInfo): Future[Option[DateTime]]
   def deleteToken(userInfo: UserInfo): Future[Unit]
+  def toProxyFromUser(userInfo: UserInfo): String
```

Comment:
This conflicts with the version already present in `HttpGoogleServicesDAO` and is also never called.

If you have a specific reason for moving these two methods to this trait, make them consistent.  Otherwise remove them.